ansible/inventory/group_vars/operator-pipeline-integration-tests.yml

```diff
@@ -24,3 +24,4 @@ nginx_proxy_url: "{{ community_signing_pipeline_name }}-nginx-{{ oc_namespace }}
 # Webhook vars
 operator_pipeline_github_webhook_repos:
   - redhat-openshift-ecosystem/operator-pipelines-test
+  - redhat-openshift-ecosystem/community-operators-pipeline-preprod
```

Comment:
Have you verified if this is really working? The webhook for isv tests has a predefined URL based on a pipeline name. The same webhook for the community has a different URL.